client/src/pages/searches/Search.js

```diff
@@ -1570,6 +1581,14 @@ const Search = ({
   function closeSaveModal() {
     setShowSaveSearch(false)
   }
+
+  function getResultsLimit() {
```

Comment:
Function can be inlined; you're calling it only once and it is a simple statement.